src/test/claimtrieupdatebuffer_tests.cpp

```diff
@@ -67,18 +68,18 @@ BOOST_AUTO_TEST_CASE(recursiveprune_test)
     base_node.insertClaim(test_claim);
 
     // node 1 has a claim so it should not be pruned
-    CClaimTrieNode node_1;
+    CClaimTrieNode *node_1 = new CClaimTrieNode;
```

Comment:
The old code looked more correct to me. Why the change to an allocation? The modern c++ says that the `new` keyword is generally deprecated. Use `std/boost::make_unique` or `make_shared` if you really need to allocate on the heap, but try to avoid it generally.

Reply:
Children are vector of shared pointers

Reply:
new is completely not deprecated or so on.